Sage/carbon
# fix(textbox): fix incorrect margin on prefix (FE-5566)

### Proposed behaviour

![image](https://user-images.githubusercontent.com/22885392/208878794-92feb1f5-dcb7-4cc8-890b-dfbb5384441b.png)


### Current behaviour

![image](https://user-images.githubusercontent.com/22885392/208876202-d42e679d-3451-47cd-8abe-0421ef71f4c9.png)

### Checklist



~~- [ ] All themes are supported if required~~
~~- [ ] Cypress automation tests added or updated if required~~
~~- [ ] Translations added or updated (including creating or amending translation keys table in storybook) if required~~
~~- [ ] Typescript `d.ts` file added or updated if required~~

#### QA


### Additional context

Fixes #5646

### Testing instructions

1. run locally
2. open `http://localhost:9001/?path=/docs/textbox-test--default`
3. fill in the `prefix` prop

The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.
